src/promptflow/promptflow/batch/_batch_inputs_processor.py

```diff
@@ -38,15 +50,22 @@ def process_batch_inputs(self, input_dirs: Dict[str, str], inputs_mapping: Dict[
                 "and consider resubmitting.\n{input_dirs}"
             )
             raise EmptyInputsData(message_format=message_format, input_dirs=input_dirs_str)
-        return self._validate_and_apply_inputs_mapping(input_dicts, inputs_mapping)
+        return input_dicts
+    def process_batch_inputs_without_inputs_mapping(self, input_dirs: Dict[str, str]):
+        input_dicts = self._resolve_input_data_and_check(input_dirs)
+        merged_list = self._merge_input_dicts_by_line(input_dicts)
+        if len(merged_list) == 0:
+            raise InputMappingError(
+                message_format=(
+                    "The input for batch run is incorrect. Could not find one complete line on the provided input. "
+                    "Please ensure that you supply data on the same line to resolve this issue."
+                )
+            )
 
-    def _resolve_input_data(self, input_dirs: Dict[str, str]):
-        """Resolve input data from input dirs"""
-        result = {}
-        for input_key, input_dir in input_dirs.items():
-            input_dir = resolve_dir_to_absolute(self._working_dir, input_dir)
-            result[input_key] = self._resolve_data_from_input_path(input_dir)
-        return result
+        return merged_list
+
+    def _process_batch_inputs_line(self, inputs: Dict[str, Any], inputs_mapping: Dict[str, str]):
```

Comment:
why we need this function given it calls another function only?

Reply:
Updated. I add additional checking before call another function.